Tests/SentryTests/Integrations/Breadcrumbs/SentrySystemEventBreadcrumbsTest.swift

```diff
@@ -218,6 +219,14 @@ class SentrySystemEventBreadcrumbsTest: XCTestCase {
         NotificationCenter.default.post(Notification(name: UIApplication.userDidTakeScreenshotNotification))
         assertBreadcrumbAction(scope: scope, action: "UIApplicationUserDidTakeScreenshotNotification")
     }
+
+    func testTimezoneChangeBreadcrumb() {
```

Comment:
`m`: I think we still miss a few tests to cover all the different edge cases.

Reply:
Absolutely, but with the test being quite flaky, it doesn't make sense to test for the correct `previous_seconds_from_gmt` and `current_seconds_from_gmt` values for example, since at the moment `previous_seconds_from_gmt` is also 0.